include/targets/cuda/constant_kernel_arg.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <util_quda.h>
+#include <quda_api.h>
+#include <target_device.h>
+
+/**
+   @file constant_kernel_arg.h
+
+   This file should be included in the kernel files for which we wish
+   to utilize __constant__ memory for the kernel parameter struct.
+   This needs to be included before the definition of the kernel,
+   e.g., kernel.h in order for the compiler to do the kernel
+   instantiation correctly.
+ */
+
+namespace quda {
+
+  namespace device {
+    
+    /**
+       @brief The __constant__ buffer used for kernel parameters
+    */
+#if defined(__CUDACC_RDC__) && !defined(QUDA_CONSTANT_DEFINE) // what is the equivalent macro for nvc++?
```

Comment:
is that still an open TODO?

Reply:
Turns out the equivalent macro is `__CUDACC_RDC__` 😄